2022 - Mechanicum.cat

```diff
@@ -2030,7 +2044,7 @@
                 <constraint field="selections" scope="parent" value="0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="8fe0-02cf-f34d-c225" type="min"/>
               </constraints>
               <entryLinks>
-                <entryLink id="6940-130e-f973-8451" name="Servo-Arm" hidden="false" collective="false" import="true" targetId="4168-fc85-8912-7188" type="selectionEntry">
+                <entryLink id="6940-130e-f973-8451" name="Servo-Arm" hidden="false" collective="false" import="true" targetId="4168-fc85-8912-7188" type="selectionEntry" defaultAmount="1">
```

Comment:
I also set this, doesn't do anything in battlescribe because the list that sets if you can take a servo-arm doesn't auto-set but in nr it sets to the first option.